client/components/app.jsx

```diff
@@ -6,7 +6,8 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      grades: []
+      grades: [],
+      averageGrade: 0
```

Comment:
This isn't being used, you can remove it.

Reply:
Forgot to remove it when I changed how to send the prop over to the Header component, fixed now